src/frontend/pages/components/BlockTest.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import Block from "../../../frontend/containers/block/index";
+import { generateTestingDocs1 } from "../../../db/testing_data";
+import * as dbTypes from "../../../db/types";
+import * as cfgTypes from "../../../cfg/types";
+import * as db from "../../../db";
+import * as queries from "../../../db/queries";
+import { LoadedDb } from "../../../db/types";
+import { useRxQuery } from "rxdb-hooks";
+
+const docs: dbTypes.DbDocument[] = generateTestingDocs1();
+
+const AllBlocks = (props: {db: LoadedDb}) => {
+  const { db } = props;
+  const { result: docs } = useRxQuery(queries.allBlocks(db.db));
+  const blocks: dbTypes.Block[] = docs.map((d) => d.get());
+
+  return (
+    <>
+      {blocks.map(block => <Block db={db} block={block} /> )}
+    </>
+  );
+};
+
+const BlockDev = () => {
```

Comment:
added custom component just for development, where we load an expected database.  This way we dont have to clock through to test.